aws-amp-integration.md

```diff
@@ -1,70 +1,154 @@
 
-## 3. Set up your Prometheus to scrape metrics from Kubecost
+### Configure Kubecost to use AMP as a time series database
 
-First, create a file called `extra_scrape_configs.yaml` with the following contents, replacing `<your_kubecost_namespace>` with your Kubecost namespace:
+If you use the default values as in this documentation, you can simply run this command to update Kubecost Helm release to use your AMP workspace as a time series database.
 
+```bash
+helm upgrade -i kubecost \
+oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
+--namespace kubecost --create-namespace \
+-f https://tinyurl.com/kubecost-amazon-eks \
+-f https://tinyurl.com/kubecost-amp \
+--set global.amp.prometheusServerEndpoint=${QUERYURL} \
+--set global.amp.remoteWriteService=${REMOTEWRITEURL}
 ```
-- job_name: kubecost
-  honor_labels: true
-  scrape_interval: 1m
-  scrape_timeout: 10s
-  metrics_path: /metrics
-  scheme: http
-  dns_sd_configs:
-    - names:
-        - kubecost-cost-analyzer.<your_kubecost_namespace>
-      type: "A"
-      port: 9003
 
-```
+For advanced configuration, you can download [values-amp.yaml](https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-amp.yaml) locally to edit it accordingly then run the following command:
 
-Then, add these recording rules as `serverFiles` to the Prometheus `override_values.yaml` override file. While they are optional, they offer improved performance.
+```bash
+helm upgrade -i kubecost \
+oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
+--namespace kubecost --create-namespace \
+-f https://tinyurl.com/kubecost-amazon-eks \
+-f PATH_TO_THE_LOCAL_DIRECTORY/values-amp.yaml
+```
 
+Next, run the following command to restart the Prometheus deployment to reload the service account configuration:
+  
+```bash
+kubectl rollout restart deployment/kubecost-prometheus-server -n kubecost
 ```
-serverFiles:
-  rules:
+
+Your Kubecost setup is now start writing and collecting data from AMP. Data should be ready for viewing within 15 minutes.
+
+---
+
+To verify that the integration is set up, check the `Prometheus Status` section on Kubecost Settings page.
+
+![Prometheus status screenshot](https://user-images.githubusercontent.com/22844059/132998278-fd388e9a-8d61-4b8b-ad1c-0e52f17ca251.png)
+
+Have a look at the [Custom Prometheus integration troubleshooting guide](https://docs.kubecost.com/custom-prom.html#troubleshooting-issues) if you run into any errors while setting up the integration. You're also welcome to [reach out to us on Slack](https://join.slack.com/t/kubecost/shared_invite/zt-1dz4a0bb4-InvSsHr9SQsT_D5PBle2rw) if you require further assistance or if you need support from AWS team, you can submit a support request through your existing [AWS support contract](https://aws.amazon.com/contact-us/).
+
+### Add recording rules (optional)
```

Comment:
@AjayTripathy do you think these should be included given their current role?

Reply:
No, I don't think they're necessary.